packages/svelte-check/src/writers.ts

```diff
@@ -14,29 +14,26 @@ export interface Writer {
 }
 
 export class HumanFriendlyWriter implements Writer {
-    _stream: Writable;
-    _isVerbose: boolean;
-
-    constructor(stream: Writable, isVerbose = true) {
-        this._stream = stream;
-        this._isVerbose = isVerbose;
+    constructor(private stream: Writable, private isVerbose = true) {
+        this.stream = stream;
```

Comment:
`this.X = X` can be omitted, ts will do that for us 😃 
So it just becomes `constructor(private stream: Writable, private isVerbose = true) {}`

Reply:
done with previous commit.

Reply:
Not quite, it's 

```
constructor(private stream: Writable, private isVerbose = true) {
        this.stream = stream;
        this.isVerbose = isVerbose;
    }
```

but can be 
`constructor(private stream: Writable, private isVerbose = true) {} // <- empty constructor body`

Or github is showing me something wrong here 😄

Reply:
Ah, sorry, misunderstood your original request. Done now.